test/mocha/10-provision.js

```diff
@@ -104,6 +105,24 @@ describe('provision API', () => {
       const {id: capabilityAgentId} = capabilityAgent;
       result.controller.should.equal(capabilityAgentId);
     });
+    it('throws error when creating a config without assertion method zcap',
+      async () => {
+        let err;
+        let result;
+        try {
+          const zcapsCopy = {...zcaps};
+          delete zcapsCopy['assertionMethod:Ed25519'];
+          result = await helpers.createConfig({
+            capabilityAgent, zcaps: zcapsCopy
+          });
+        } catch(e) {
+          err = e;
+        }
+        should.exist(err);
+        should.not.exist(result);
+        err.data.name.should.equal('OperationError');
+        err.data.message.should.equal('An unspecified error occurred.');
```

Comment:
NOTE: The error message isn't the same as the one being thrown here https://github.com/digitalbazaar/bedrock-vc-issuer/blob/e7ea867f8c4f9f9e48035279a38b3f753eb56df5/lib/suites.js#L39. This error gets caught and re-thrown here https://github.com/digitalbazaar/bedrock-service-core/blob/main/lib/http/configs.js#L91. I think this should probably be changed to a BedrockError. Is that correct?